### PR TITLE
Fix(bidi): Fix for bug /codemirror/CodeMirror/issues/4897

### PR DESCRIPTION
**Pull request by wajnberg**
Fix for bug https://github.com/codemirror/CodeMirror/issues/4897

### CODE REVIEW

**Comment by wajnberg** [comment]
@TOMERM FYI

**Comment by tomerm** [comment]
Thanks, @wajnberg.  @amirbrans FYI.

File: lib/codemirror.css [hunk 1]
```diff
@@ -270,6 +270,7 @@ div.CodeMirror span.CodeMirror-nonmatchingbracket {color: #f22;}
 .CodeMirror-widget {}
 
 .CodeMirror-rtl pre { direction: rtl; }
+.CodeMirror-ltr pre { direction: ltr; }
```
Comment (marijnh) [inline lib/codemirror.css]: I think you can just style `.CodeMirror pre`, _before_ the other rule, and don't need an additional class. Would that work? I'd prefer not to have another always-present class on the main container.

**Comment by wajnberg** [comment]
Hello @marijnh 

I removed the rtl class following your advice.

Could you please review ?

Thank  you

**Comment by marijnh** [comment]
If I set the outer page to RTL and the editor to LTR in Chrome, I'm seeing an extra, badly behaving scroll bar on the left of the editor, overlapping the gutter. That could be fixed by moving the default `direction: rtl` to the top container (`.CodeMirror`). Does that sound reasonable?

(Eventually, we'd like to support scrollbars on the left, but we're not there yet.)

**Comment by wajnberg** [comment]
Hello @marijnh ,

You are right. Sorry I didn't notice that. I made the change you requested.

Could you please review ?

Thank you

**Comment by marijnh** [comment]
Much better! Merged.

**Comment by wajnberg** [comment]
Hello @marijnh ,

Thank you very much for having merged this change
Could you please tell me in which CodeMirror release this fix would be available?

Thank you

**Comment by marijnh** [comment]
The next release (5.29.0) will be out in two weeks (I do regular releases monthly, around the 20th)

**Pull request merged**